.github/ISSUE_TEMPLATE/cherry-pick-request.yml

```diff
@@ -0,0 +1,158 @@
+    label: Pull Request(s)
+    description: The PR(s) that fix the bug. Make sure they are merged and have passing [CI builds](https://app.circleci.com/pipelines/github/ampproject/amphtml?branch=main).
+    placeholder: "e.g. #22222, #33333"
+  validations:
+    required: true
+- type: input
+  id: release_tracker
+  attributes:
+    label: Release Tracker
+    description: The [tracker issue](https://github.com/ampproject/amphtml/labels/Type%3A%20Release) for the release to which the cherry-pick will be applied. Remember to update the issue title with this info.
+    placeholder: "e.g. #44444"
+  validations:
+    required: true
+- type: dropdown
+  id: channels
+  attributes:
+    label: Channels
+    description: The [release channels](https://github.com/ampproject/amphtml/blob/main/docs/release-schedule.md#release-channels) to which the cherry-pick will be applied. Remember to update the issue labels with this info.
+    multiple: true
+    options:
+      - Beta / Experimental
+      - Stable
+      - LTS
+  validations:
+    required: true
+- type: markdown
+  id: channel_instructions
+  attributes:
+    value: |
+      Select one or more channel(s) to which this cherry-pick is applicable.
+      - A cherry-pick to Stable will most likely require a cherry-pick to Beta / Experimental as well.
+      - Without this, it is possible that the fix could be lost when Beta goes to Stable.
+      - Once you select a set of channels, make sure the issue labels match your selection.
+- type: textarea
+  id: justification
+  attributes:
+    label: Justification
+    description: Why you believe this issue meets the [cherry-pick criteria](https://github.com/ampproject/amphtml/blob/main/docs/contributing-code.md#Cherry-picks).
+    placeholder: Provide a specific rationale for why this fix cannot wait until the next Stable release.
+  validations:
+    required: true
+- type: textarea
+  id: why_not_beta_experimental
+  attributes:
+    label: Why Not Beta / Experimental?
+    description: Why a Stable cherry-pick is necessary, but not Beta / Experimental.
+    placeholder: If the cherry-pick is going into Stable but not Beta / Experimental, provide a rationale (e.g. the PR is already part of the beta channel). Otherwise, skip this section.
+- type: textarea
+  id: why_lts
+  attributes:
+    label: Why LTS?
+    description: Why an LTS cherry-pick is necessary.
+    placeholder: If the cherry-pick is also going into LTS, provide a rationale (e.g. the bug is visible to LTS users). Otherwise, skip this section.
+- type: textarea
+  id: verification_steps
+  attributes:
+    label: Verification Steps
+    description: How the fix can be verified once the cherry-pick has been performed.
+    placeholder: Provide detailed steps to manually verify the changes in this cherry-pick. They will be run by a developer or a QA expert to ensure that the bug was indeed fixed.
+  validations:
+    required: true
+- type: textarea
+  id: cherry_pick_progress
```

Comment:
Move to bottom to keep most important questions for the CP requester up top, or
 
Remove this to keep the form concise, and let release engineer update the issue with comments, which flow chronologically and is easy to follow.

Reply:
The reason I added this section in the first place was because several past CP issues are missing progress updates and still show as pending (stale title, lack of comments, etc.). Checking a checkbox that's already present is far less trouble, and can be done by the requester too.

Agree with your point about keeping important questions up top. Moved this to the bottom.